webapp/src/components/data_selectors/backend_selector.tsx

```diff
@@ -160,7 +160,15 @@ export default class BackendSelector extends React.PureComponent<Props, State> {
 
             // option's label hasn't been fetched yet
             return {
-                label: v,
+                label: (
+                    <span>
+                        <img
+                            src={v.avatarUrls[AvatarSize.SMALL]}
+                            style={{width: '24px', marginRight: '10px'}}
+                        />
+                        <span>{v.displayName}</span>
+                    </span>
+                ),
```

Comment:
Hi @sanjaydemansol, I'm not sure how this fixes the issue, mainly because I don't know what the problem originally was that was causing the crash. Do you mind explaining what the problem is, and how this PR solves it?

Reply:
This component `backend_selector.tsx` is meant for general purpose autocomplete functionality, so we shouldn't have anything to do with users or avatars in this file. Why are you thinking this fix needs to go here?

Reply:
There was `v` assigned to label and that was an object, because of that it was not able to render it properly.

Reply:
@sanjaydemansol I'm thinking the assignee-specific change needs to be applied to https://github.com/mattermost/mattermost-plugin-jira/blob/master/webapp/src/components/data_selectors/jira_user_selector/jira_user_selector.tsx. It makes sense if there's also a small change needed in `backend_selector.tsx`, but we shouldn't be referencing anything to do with avatars in this file.

Reply:
@mickmister code updated, removed avatar and code is not generating any errors.

Reply:
> code is not generating any errors.

@sanjaydemansol To be clear: Before your fix, the webapp was _crashing_ correct? And after your fix, it is not crashing.